README.md

```diff
@@ -47,7 +47,7 @@ Shipwright supports any tool that can build container images in Kubernetes clust
 - Install the Shipwright deployment. To install the latest version, run:
 
   ```bash
-  kubectl apply --filename https://github.com/shipwright-io/build/releases/download/v0.12.0/release.yaml --server-side
+  kubectl apply --filename https://github.com/shipwright-io/build/releases/download/v0.13.0/release.yaml --server-side
   curl --silent --location https://raw.githubusercontent.com/shipwright-io/build/v0.12.0/hack/setup-webhook-cert.sh | bash
```

Comment:
@SaschaSchwarze0 looks like our GitHub action is missing a few spots where the prior release is referenced.

Reply:
https://github.com/shipwright-io/build/issues/1601